decidim-core/app/cells/decidim/profile_actions_cell.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ProfileActionsCell < RedesignedProfileCell
+    include CellsHelper
+    include Decidim::Messaging::ConversationHelper
+
+    delegate :user_signed_in?, to: :controller
+
+    ACTIONS_ITEMS = {
+      edit_profile: { icon: "pencil-line", path: :profile_edit_path },
```

Comment:
These are overflowing on a narrow screen, e.g. "Galaxy Fold" that is available in the Chrome device toolbar:
![Overflowing buttons](https://user-images.githubusercontent.com/864340/187220344-16770a25-3c61-4905-99e0-de466d79c564.png)

This happens on a group profile.

This is kind of a problem with that particular phone (Galaxy Fold) when the phone is in folded mode. A potential fix is explained here:
https://r00t.io/managing-websites/a-way-to-fix-websites-for-samsung-galaxy-folds-280px-width-display/

There they mention that even their own landing page for that phone did not work on that screen width. :smile:

Reply:
Moved to #9670.